clevertap-core/src/main/java/com/clevertap/android/sdk/CTXtensions.kt

```diff
@@ -299,3 +305,17 @@ fun String?.toJsonOrNull(): JSONObject? {
         }
     }
 }
+
+fun View.applySystemBarsInsetsWithMargin(block : (bars:Insets, mlp:MarginLayoutParams) -> Unit) {
+    ViewCompat.setOnApplyWindowInsetsListener(this
+    ) { v, insets ->
+        val bars: Insets = insets.getInsets(
+            WindowInsetsCompat.Type.systemBars() or WindowInsetsCompat.Type.displayCutout()
```

Comment:
System bars are fine, not sure what is `displayCutout()` , please make sure this is needed.

Reply:
To make UI compatible with top and bottom notches(displayCutout)